packages/components/atoms/f-form-field/stories/Textarea.stories.js

```diff
@@ -0,0 +1,75 @@
+import {
+    withKnobs, select, text, boolean
+} from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
+import FormField from '../src/components/FormField.vue';
+import {
+    DEFAULT_FIELD_SIZE,
+    VALID_FIELD_SIZES,
+    VALID_TEXT_INPUT_TYPES
+} from '../src/constants';
+
+export default {
+    title: 'Components/Atoms/f-form-field',
+    decorators: [withKnobs, withA11y]
+};
+
+export const TextareaComponent = () => ({
+    components: { FormField },
+    props: {
+        locale: {
+            default: select('Locale', ['en-GB', 'en-AU'])
+        },
+        labelText: {
+            default: text('Label Text', 'First name')
+        },
+        inputType: {
+            default: select('Input Type', VALID_TEXT_INPUT_TYPES)
+        },
+        isDisabled: {
+            default: select('isDisabled', [null, 'disabled'])
+        },
+        shouldShowLabelText: {
+            default: boolean('shouldShowLabelText', true)
+        },
+        hasError: {
+            default: boolean('hasError', false)
+        },
+        isGrouped: {
+            default: boolean('isGrouped', false)
+        },
+        labelDescription: {
+            default: text('Label Description', '')
+        },
+        labelDetails: {
+            default: text('Label Details', '')
+        },
+        assistiveText: {
+            default: text('Assistive Text', '')
+        },
+        placeholder: {
+            default: text('Placeholder', '')
+        }
+    },
+    parameters: {
+        notes: 'some documentation here'
+    },
+    template:
+        `<form-field
```

Comment:
Should be `'` instead of <code>`</code>.

Reply:
Same as above.